vunit/ui/__init__.py

```diff
@@ -157,7 +159,14 @@ def test_filter(name, attribute_names):
 
         self._builtins = Builtins(self, self._vhdl_standard, simulator_class)
         if compile_builtins:
-            self.add_builtins()
+            self.add_vhdl_builtins()
+            builtins_deprecation_note = (
+                "'compile_builtins' (which defaults to 'True') is deprecated "
+                "and it will be removed in future releases; "
+                "preserve the functionality using "
+                "'compile_builtins=False' and 'VU.add_vhdl_builtins'"
+            )
```

Comment:
This message should mention that `compile_builtins` is related to the `argv` and `args` methods. Most people probably don't know about its existence. A link to the docs would also be he helpful.

Reply:
I made the note more visible and explicit. I also included references to the methods in the docs.

After we merge this PR, I think we should create a pinned issue, and provide specific code examples (such as the ones below) for versions before 4.7.0, during 4.70 and after 4.7.0.